swim/labels_test.go

```diff
@@ -1,12 +1,98 @@
 package swim
 
 import (
+	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+var (
```

Comment:
Can you add a test which fully initializes ringpop with much smaller configured limits, and verify that they are honored?